gateway-manager/src/zookeeper_functions.py

```diff
@@ -226,6 +226,8 @@ def make_user(zk, name, pw):
 
 
 def get_zookeeper():
+    if not ZK_HOST:
+        raise ValueError('Failure: ZOOKEEPER_HOST must set.')
```

Comment:
https://docs.python.org/3/library/exceptions.html#ValueError
:thinking: 

For me, this is more like a `RuntimeError` or even better `EnvironmentError`
https://docs.python.org/3/library/exceptions.html#RuntimeError